pkg/entrypoint/entrypointer.go

```diff
@@ -100,12 +109,39 @@ func (e Entrypointer) Go() error {
 	// Write the post file *no matter what*
 	e.WritePostFile(e.PostFile, err)
 
+	if e.Results != nil {
+		if err := e.handleResultFiles(); err != nil {
+			logger.Fatalf("Error while handling results: %s", err)
+		}
+	}
 	if wErr := termination.WriteMessage(e.TerminationPath, output); wErr != nil {
 		logger.Fatalf("Error while writing message: %s", wErr)
 	}
 	return err
 }
 
+func (e Entrypointer) handleResultFiles() error {
+	output := []v1alpha1.PipelineResourceResult{}
+	for _, resultFile := range e.Results {
+		// check result file
+		fileContents, err := ioutil.ReadFile(DefaultResultPath + resultFile)
```

Comment:
suggest using `filepath.Join(DefaultResultPath, resultFile)` here to account for operating system specific separators.